src/functionalTest/java/uk/gov/hmcts/reform/opal/config/DBConnection.java

```diff
@@ -0,0 +1,38 @@
+package uk.gov.hmcts.reform.opal.config;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class DBConnection {
```

Comment:
Do we need this class at all? As it's always trying to connect to local host, I don't see this is required. Please remove this.